code/modules/power/apc.dm

```diff
@@ -726,6 +757,21 @@
 				spark_system.queue()
 				H << "<span class='danger'>The APC power currents surge eratically, damaging your chassis!</span>"
 				H.adjustFireLoss(10, 0)
+			if(infected)
+				if(H in hacked_ipcs)
+					return
+				LAZYADD(hacked_ipcs, "\ref[H]")
+				infected = 0
+				H << "<span class = 'danger'>Fil$ Transfer Complete. Er-@4!#%!. New Master detected: [hacker]! Obey their commands.</span>"
+				hacker << "<span class = 'notice'> Corrupt files transfered to [H]. They are now under your control. This will not last long.</span>"

```

Comment:
You have odd spaces between span closing markers and the start of sentences. Please look over these and remove them where necessary.

`"<span class = 'notice'> Corrupt files transfered to [H]. They are now under your control. This will not last long.</span>"` should be `"<span class = 'notice'>Corrupt files transfered to [H]. They are now under your control. This will not last long.</span>"`